docs/servo_power_monitor/src/index.ts

```diff
@@ -1,46 +1,96 @@
 // while true ; do { echo "do nothing for 5 sec" ; sleep 5 ; echo "yes for 5 sec
 // without displaying" ; timeout 5 yes > /dev/null ; } ; done ectool
 // chargecontrol idle ectool chargecontrol normal
-
-import {
-  analyzePowerData,
-  handleFileSelect,
-  handleDragOver,
-  executeScript,
-  formSubmit,
-  selectDutSerial,
-  requestSerial,
-  disconnectSerialPort,
-  downloadJSONFile,
-  stopMeasurement,
-  cancelSubmit,
-} from './main';
-import {
-  analyzeAddClickEvent,
-  downloadAddClickEvent,
-  dropZoneAddDragoverEvent,
-  dropZoneAddDropEvent,
-  executeScriptAddClickEvent,
-  formAddSubmitEvent,
-  haltAddClickEvent,
-  inputAddKeydownEvent,
-  requestSerialAddClickEvent,
-  selectDutSerialAddClickEvent,
-  setPopupCloseButton,
-} from './ui';
+import moment from 'moment';
+import {OperatePort} from './operatePort';
```

Comment:
https://google.github.io/styleguide/tsguide.html#identifiers-imports

File names should be snake_case